### DISCUSSION TITLE
How to install the specific version of linkerd

### DISCUSSION THREAD

**Discussion by protim1993** [linkerd/linkerd2#7909] [Q&A]
I have linkerd 2.11 CLI running in my system but I need to run the 2.9.4 stable version in my local machine. Can someone assist how to install the specific version of LinkerD

**Comment by alpeb** [comment]
You should just pick the right binary from the releases page ([here](https://github.com/linkerd/linkerd2/releases/tag/stable-2.9.4) for 2.9.4) and run that binary directly.

**Comment by wmorgan** [comment]
You can also use the LINKERD2_VERSION environment variable with the installer, e.g.:

`curl --proto '=https' --tlsv1.2 -sSfL https://run.linkerd.io/install-edge | LINKERD2_VERSION=edge-25.1.1 sh`
